zerver/openapi/zulip.yaml

```diff
@@ -9234,6 +9234,20 @@ paths:
                           array will be empty if `enable_drafts_synchronization` is set to `false`.
                         items:
                           $ref: "#/components/schemas/Draft"
+                      emoji_data_url:
```

Comment:
If a client wants this, it should include 'emoji_data_url' in `fetch_event_types`, right? (Or `event_types`? Mm, probably not that.)

Reply:
We could decide this is part of the state you get regardless as part of the general server data, or bundle it with "if you asked for `realm_emoji`". Though I suppose if we're calling it part of the server data, we may want to name it `server_emoji_data_url` so that it's clear it should go into a `server` dictionary if we decide to adjust the scope. Right now those are bundled in `want("realm")`, and that's probably a reasonable place for them.

Reply:
So to be speciifc, I propose:
`server_emoji_data_url` that's included if `want("realm")` as likely the best option. This would parallel, for example, `server_web_public_streams_enabled`.